src/routes/docs/advanced/platform/otp-sms/+page.markdoc

```diff
@@ -0,0 +1,221 @@
+| 677          | Solomon Islands                                              | 0.051       |
+| 252          | Somalia                                                      | 0.1222      |
+| 27           | South Africa                                                 | 0.0215      |
+| 34           | Spain                                                        | 0.0402      |
+| 94           | Sri Lanka                                                    | 0.2613      |
+| 290          | St. Helena                                                   | 0.0416      |
+| 249          | Sudan                                                        | 0.2015      |
+| 597          | Suriname                                                     | 0.0715      |
+| 268          | Swaziland                                                    | 0.0872      |
+| 46           | Sweden                                                       | 0.0443      |
+| 41           | Switzerland                                                  | 0.0325      |
+| 963          | Syria                                                        | 0.195       |
+| 886          | Taiwan                                                       | 0.1851      |
+| 66           | Thailand                                                     | 0.0161      |
+| 228          | Togo                                                         | 0.1001      |
+| 676          | Tonga                                                        | 0.0295      |
+| 216          | Tunisia                                                      | 0.0805      |
+| 90           | Turkey                                                       | 0.0065      |
+| 688          | Tuvalu                                                       | 0.0912      |
+| 256          | Uganda                                                       | 0.1033      |
+| 380          | Ukraine                                                      | 0.11        |
+| 971          | United Arab Emirates                                         | 0.0754      |
+| 44           | United Kingdom                                               | 0.0338      |
+| 598          | Uruguay                                                      | 0.0442      |
+| 678          | Vanuatu                                                      | 0.0912      |
+| 58           | Venezuela                                                    | 0.0585      |
+| 84           | Vietnam                                                      | 0.0949      |
+| 967          | Yemen                                                        | 0.1476      |
+| 260          | Zambia                                                       | 0.1154      |
+| 255          | Zanzibar                                                     | 0.0872      |
+| 263          | Zimbabwe                                                     | 0.0872      |
+
+# Improving reliability {% #improving-reliability %}
```

Comment:
This entire section is really not needed. Adds no value. What we wanted to add instead, and can be combined in the section above is that we're constantly working with telecom providers to reduce the SMS rates as much as possible in effort to make services as affordable as possible.